library/src/main/java/com/okta/appauth/android/OAuthClientConfiguration.java

```diff
@@ -194,7 +194,7 @@ void readConfiguration(@NonNull final JSONObject jsonObject)
             throws InvalidJsonDocumentException {
         JsonParser jsonParser = JsonParser.forJson(jsonObject);
 
-        //We can not take hash code directly from JSONObject
+        //We can not take has code directly from JSONObject
```

Comment:
typo: `hash`

Reply:
Done but in pull requests with token revocation